### ISSUE TITLE
Groups tree not saved correctly

### ISSUE THREAD

**Issue by bernhard-kleine** [JabRef/jabref#2779]
JabRef 4.0.0-beta (from April 20, 2017)
Windows 7 6.1 amd64 
Java 1.8.0_121

Steps to reproduce:

1. Open jabref 4.00-beta...
2. the groups are gone
3. ...

![screenjabrefgroupsgone](https://cloud.githubusercontent.com/assets/9216487/25286580/8550534a-26be-11e7-9a2c-f6645660f37d.PNG)

Actually, they were there when I opened the file with jabref. I then edited some groups titles and saved the file. I wanted to check what was displayed in the snapshot I have been working before. The groups in the group list were no longer displayed, while the group entry was shown. Strange! I killed the file and reopened: Now the All entries does only count 82 while the entry list shows 726 entries. Very strange

Any proposal?

**Comment by Siedlerchr** [comment]
This is really strange.
Is there anything in the log window? An exception?

Am 21.04.2017 6:47 nachm. schrieb "Bernhard Kleine" <
notifications@github.com>:

**Comment by bernhard-kleine** [comment]
I have the file FatMetabolism.bib with 727 entries. with jabref 4.0.0 beta a groups tree can not be added to this file. 
1. I added two groups to the file, one empty and one with 87 entries. 
2. I save the bib-file.
3. I opened the bib-file in notepad++.
4. The groupstree cannot be found.
![screenjabrefgroupsgone2](https://cloud.githubusercontent.com/assets/9216487/25294620/543ba04e-26e0-11e7-8534-c469e4ff5f00.PNG)
